openshift/templates/api/api-deploy.bcreg.param

```diff
@@ -5,13 +5,14 @@
 #=========================================================
 NAME=api
 APP_NAME=issuer
+DB_APP_NAME=issuer-kit
 APP_GROUP=bcreg
 SUFFIX=-bcreg
 ROLE=controller
 IMAGE_NAMESPACE=a99fd4-tools
 NAMESPACE_NAME=a99fd4
 SERVICE_PORT=8080
-DB_SERVICE=db
+DB_SERVICE=issuer-kit-db
```

Comment:
Can we have `DB_APP_NAME` and `DB_SERVICE` besides each other, since they are tightly coupled parameters that should always be tweaked together? Not an issue with the config, more of an optimization for maintenance purposes.

Reply:
moved params around and added DB_SUFFIX